bundles/org.eclipse.jface/src/org/eclipse/jface/action/StatusLineManager.java

```diff
@@ -194,6 +211,9 @@ public void subTask(String name) {
 
 			@Override
 			public void worked(int work) {
+				if (!taskStarted) {
```

Comment:
I'm not sure if this is covered by the API that one *must* have started the monitor before reporting work (even though it seems useful), maybe just ignore that call then? or simply store the work amount and apply it once the monitor is started...

Reply:
IProgressMonitor:25
```
When a task begins, a <code>beginTask(String, int)
 * </code> notification is reported, followed by any number and mixture of 
 * progress reports (<code>worked()</code>)
```